about.html

```diff
@@ -0,0 +1,195 @@
+            <h2 class="special-color text-center">"Who are we?"</h2>
+            <h1 class="special-color text-center">
+              Asian Youth Program (AYP) CLUB ACTIVITIES
+            </h1>
+          </div>
+          <div class="header-content text-center margin-top-big">
+            <p>
+              Asian Youth Program (AYP), in partnership with Womankind (formerly New York Asian Women’s Center), offers mentoring to children impacted by domestic violence.
+            </p>
+          </div>
+          <div class="infos">
+            <p class="location light-black text-center">
+              <i
+                >Please Contact Us with Email for any further questions about
+                this activity!</i
+              ><br /><b>AYP@gmail.com</b>
+            </p>
+          </div>
+        </div>
+      </div>
+    </header>
+    <section class="about-us flex-column-center text-center" id="about-us">
+      <h2 class="about-us-head light-black heading-medium margin-top-big">
+        About Us
+      </h2>
+      <hr class="horizontal-line text-center" />
+      <img
+        src="https://securityconference.org/assets/media/e/5/csm_themen_cyber-security-and-technology_header_compr_2037a5dc78.jpg"
+        alt="about us community photo"
+        class="family-photo"
+      />
+      <div class="about-us-content">
+        <p class="about-us-paragraph">
+          Asian Youth Program (AYP), in partnership with Womankind (formerly New York Asian Women’s Center), offers mentoring to children impacted by domestic violence.
+        </p>
+        <h4>
+          <br />
+          What do we promise to youths and adults?
+        </h4>
+        <p>
+          <br />
+          Asian Youth Program (AYP), in partnership with Womankind (formerly New York Asian Women’s Center).
+        </p>
+      </div>
+      <hr class="full-horizontal-line" />
+    </section>
+    <section class="past-activities" id="past-activities">
+      <h2 class="past-heading light-black heading-medium">
+        See the past Club Activities
+      </h2>
+      <hr class="horizontal-line text-center" />
+      <p class="past-content light-black text-center">
+        Take a look at the last two years activities which took place in
+        Asia
+      </p>
+      <div class="activities">
+        <div class="past-first-activity act">
+          <div class="first-activity-content flex-column-center">
+            <h3>2021</h3>
+            <p>AYP Summit in Asia</p>
+          </div>
+        </div>
+        <div class="past-second-activity act">
+          <div class="first-activity-content flex-column-center">
+            <h3>2020</h3>
+            <p>AYP Summit in Asia</p>
+          </div>
+        </div>
+      </div>
+      <hr class="full-horizontal-line" />
+    </section>
+    <footer class="footer flex-column-center" id="contact">
+      <h3 class="contact light-black heading-medium">Contact</h3>
+      <hr class="horizontal-line text-center" />
```

Comment:
- [ ] Great work making the changes requested by the previous reviewers. To make your project complete, kindly implement the final change requested by the previous reviewers by making the `background-color` of the `footer` tag with class `.footer` on the about page `#272a31`. After doing this, you can adjust the `color` of the fonts to be a bright `color` for accessibility purposes. Implementing this change will ensure that your project is identical to that of the template provided by Cindy Shin. Kindly implement this change 👍